docs/qcfractal/source/setup_quickstart.rst

```diff
@@ -0,0 +1,295 @@
+
+    nohup qcfractal-manager --config-file <path to config YAML> --verify=False &
+
+Note that TLS certificate verification is disabled (``--verify=False``) because the :term:`Manager` and :term:`Server` are both run on the head node.
+
+Test if everything is setup by running a Hartree-Fock calculation a single hydrogen molecule,
+as in the :doc:`quickstart` (note this requires ``psi4``):
+
+.. code-block:: python
+
+   python
+
+   >>> import qcfractal.interface as ptl
+
+   # Note that server TLS verification is turned off (verify=False) since all components are run locally.
+   >>> client = ptl.FractalClient(address="localhost:7777", verify=False)
+   >>> mol = ptl.Molecule(symbols=["H", "H"], geometry=[0, 0, 0, 0, 5, 0])
+   >>> mol_id = client.add_molecules([mol])[0]
+   >>> r = client.add_compute("psi4", "HF", "STO-3G", "energy", None, [mol_id])
+
+   # Wait a minute for the job to complete
+   >>> proc = client.query_procedures(id=r.ids)[0]
+   >>> print(proc)
+   <ResultRecord(id='0' status='COMPLETE')>
+   >>> print(proc.properties.scf_total_energy)
+   -0.6865598095254312 
+
+
+.. _quickstart-shared-cluster:
+
+Shared Clusters, Supercomputers, and Multiple Clusters
+++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+This quickstart guide addresses QCFractal setup on one or more shared cluster(s). 
+The :term:`Server` should be set up on a persistent server for which you have permission to expose ports. 
+For example, this may be a dedicated webserver, the head node of a private cluster, or a cloud instance.
+The :term:`Manager` should be set up on each shared cluster. 
+In most cases, the :term:`Manager` may be run on the head node; 
+contact your system administrator if you are unsure.
+This guide requires `Parsl <https://parsl.readthedocs.io/en/stable/quickstart.html>`_ to be installed for the :term:`Manager`. It may be installed with ``pip``.
+
+Begin by initializing the :term:`Server` on your persistent server::
+
+    qcfractal-server init 
+
+The QCFractal server receives connections from :term:`Managers <Manager>` and clients on TCP port 7777. 
+You may optionally specify the ``--port`` option to choose a custom port. 
+You may need to configure your firewall to allow access to this port.
+
+Because the :term:`Server` will be exposed to the internet, 
+security should be enabled to control access. 
+Enable security by changing the YAML file (default: ``~/.qca/qcfractal/qcfractal_config.yaml``)
+``fractal.security`` option to ``local``::
+
+   - security: null
+   + security: local
+
+Start the :term:`Server`::
+
+   nohup qcfractal-server start &
+
+You may optionally provide a TLS certificate to enable host verification for the :term:`Server` 
+using the ``--tls-cert`` and ``--tls-key`` options. 
+If a TLS certificate is not provided, communications with the server will still be encrypted, 
+but host verification will be unavailable 
+(and :term:`Managers <Manager>` and clients will need to specify ``--verify False``).
```

Comment:
`verify=False` not `--verify`. Maybe add a line here that this is ok since you will also be connecting to a server that you trust.